src/openms/source/FORMAT/PepXMLFile.cpp

```diff
@@ -71,6 +74,8 @@ namespace OpenMS
 
   void PepXMLFile::store(const String& filename, std::vector<ProteinIdentification>& protein_ids, std::vector<PeptideIdentification>& peptide_ids, const String& mz_file, const String& mz_name, bool peptideprophet_analyzed)
   {
+    bool use_original_spectrum_name = false;
```

Comment:
@hendrikweisser if we turn this on (or at least make it an option) it would allow us to retain the original `<spectrum_query spectrum="hroest_K120718_SM_OGE10_010_IDA.02552.02552.2"` information when writing out pep.xml -> personally I would prefer that but it might break some existing code which basically uses `PepXMLFile_test.00000.00000.3` which to me seems inferior

is there some code that relies on the OpenMS `0000.0000` version?